lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js

```diff
@@ -478,32 +477,45 @@
 
       brush.on("brushstart.chart", function() {
         var extent = brush.extent();
+        initialBounds = [extent];
```

Comment:
We store the initial unmodified (`by brush.extent()`) extent here, and then pass on updating the UI in the first `brush.chart` event callback in order to save the extent after its been modified by `brush.extent()`. Then, in subsequent `brush.chart` callbacks, we compare the current extent value to the modified initial extent, to determine if only one edge is changing. If only one is, we reset the unchanging edge to the original extent value to keep the unchanging edge from doing any weird snapping.

Pretty hacky but I can't figure out another way to determine if only one edge is changing. 

This all goes out the window if we try to round the extents in the callbacks to the nearest histogram bin, because then it is ambiguous whether only one or both bounds are changing in some dragging cases. So that was removed for now, unless we can find another solution.